src/services/inlayHints.ts

```diff
@@ -730,6 +722,9 @@ export function provideInlayHints(context: InlayHintsContext): InlayHint[] {
                     const propertySignature = node as PropertySignature;
                     if (propertySignature.modifiers) {
                         visitDisplayPartList(propertySignature.modifiers, " ");
+                        if (propertySignature.modifiers.length) {
+                            parts.push({ text: " " });
+                        }

```

Comment:
```suggestion
                    if (propertySignature?.modifiers.length) {
                        visitDisplayPartList(propertySignature.modifiers, " ");
parts.push({ text: " " });
```

Doesn't make sense to visit the list if there are no modifiers.